### ISSUE TITLE
❌ payment

### ISSUE THREAD

**Issue by proteanads** [elementor/elementor#20988]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

it is stugg at the time of wordpress settings page into payment button click

### Describe the solution you'd like

reply for soonest solution of my issue

### Describe alternatives you've considered

i am awating for solution

**Comment by nicholaszein** [comment]
Hey @proteanads, 

Thank you for submitting your issue! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Clear and concise title:** Please provide a clear and concise title that gives context to your issue;
- **Prerequisites:** please search the repository before submitting an issue;
- **Description:** In your description, you need to explain in detail what happened, as you would explain to someone that doesn't know anything about your website or Elementor. Give as much information as possible. This is important information that will help solve your issue quicker;
- **Steps to replicate:** Think of it as explaining to someone over the phone how to replicate your issue. They can't see the problem, so explain it in detail in a way that we can try to replicate your issue in our own environments. Be detailed;
- **Isolating the Problem:** You need to make sure this only happens while using the latest versions of Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** Please include the [System Info](https://elementor.com/help/systems-info/) report as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) available in your _WordPress Dashboard > Elementor > System Info_. This information is crucial for us to create an exact copy of your environment and validate your issue. Watch [this video](https://www.youtube.com/watch?v=tDXdynsnYuA) to learn how to download the report. We can't accept issues without it.
- **One issue per thread:** We can't track multiple issues per thread. Open a new issue for each subject.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.


#### For  💡 **New Ideas** or 🚀 **Feature Requests** please [create a new Discussion](https://github.com/elementor/elementor/discussions/new) in the [Discussions](https://github.com/elementor/elementor/discussions) tab.

Best regards
